library/lua/gui/dwarfmode.lua

```diff
@@ -48,7 +48,7 @@ function enterSidebarMode(sidebar_mode, max_esc)
     local focus_string = ''
     while remaining_esc > 0 do
         local screen = dfhack.gui.getCurViewscreen(true)
-        focus_string = dfhack.gui.getFocusString(screen)
+        focus_string = dfhack.gui.getFocusStrings(screen)
```

Comment:
this clearly will not work, but this entire function is not necessary anymore and can be deleted, so it doesn't matter.